Makefile

```diff
@@ -42,7 +29,7 @@ py-unit:
 	hack/make-rules/py-test/all.sh
 .PHONY: py-unit
 image-build:
```

Comment:
nit: Rename this too since it only builds Prow images, nothing in the `images` directory.

Reply:
will do in follow up PR